### DISCUSSION TITLE
An unknown scope was requested

### DISCUSSION THREAD

**Discussion by CoolDev0723** [amzn/ads-advanced-tools-docs#177] [Q&A]
error=invalid_scope

https://apac.account.amazon.com/ap/oa?client_id=amzn1.application-oa2-client.bcd29acb405f44db9121b9b161d90188&scope=advertising::campaign_management&response_type=code&redirect_uri=https://amazon.com

Auth grant login
I tried to run this endpoint to get the code, but all I got was an error.
Please share your solution.
Thanks.

**Comment by AdsAPIPM** [comment]
you likely skipped a step in the process! This happened to me too. I have a separate thread on this https://github.com/amzn/ads-advanced-tools-docs/discussions/161#discussioncomment-7298412
